src/forms/previousExpenses.js

```diff
@@ -101,11 +95,11 @@ const Housing = function ( props ) {
       <FormHeading>Shelter</FormHeading>
 
       <MassiveToggle name={ time + 'Homeless' } value={ wasHomeless } storeChecked={ storeChecked }
-          label={'Was the household homeless at the last benefit assessment?'}/>
+          label='Was the household homeless at the last benefit assessment?' />
```

Comment:
This is more of a question of whether we want to keep things in brackets or not. There are a at least two reasons I can see for keeping things in brackets. First is consistency, second is flexibility. Consistency makes it easier to just automatically do the same thing over and over again. The code writer doesn't have to stop and remember which way they're supposed to do it. It also allows a reader to not stop to question the methodology being used. As far as flexibility goes, with brackets it's easy to add things like `someVar +` when needed without any other maneuvering. Maybe a good topic for discussion.

Reply:
To clarify, I'm not sure this PR is the place to do this and it may need further discussion before we go after this change.